fetch/main.js

```diff
@@ -0,0 +1,12 @@
+fetch('https://jsonplaceholder.typicode.com/users', {
+  method: 'GET'
+})
+  .then(response => response.json())
+  .then(data => console.log(data));
+
+fetch('https://pokeapi.co/api/v2/pokemon/tyranitar', {
+  method: 'GET'
+
+})
+  .then(response => response.json())
+  .then(data => console.log(data));
```

Comment:
Since `fetch` requests always have a chance to fail, please make sure you're always handling that failure.